src/test/java/com/hermes/recruitment/RecruitmentApplicationTests.java

```diff
@@ -0,0 +1,15 @@
+/*
+package com.hermes.recruitment;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class RecruitmentApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
+
+}
+*/
```

Comment:
Don't commit commented code
Or if you do, add a comment with a valid reason for commenting